components/Slider/README.md

```diff
@@ -215,3 +220,34 @@ id<MDCColorScheming> colorScheme = [[MDCSemanticColorScheme alloc] init];
 ```
 <!--</div>-->
 
+
+## Accessibility {a11y}
+
+<!-- Extracted from docs/accessibility.md -->
+
+## Accessibility {a11y}
+
+To help ensure your slider is accessible to as many users as possible, please be sure to review the following recommendations:
+
+### Set `-accessibilityLabel`
+
+Set an appropriate `accessibilityLabel` value for your slider. This should reflect what the slider affects.
+
+#### Objective - C
+```objc
+slider.accessibilityLabel = @"Volume Slider";
+``` 
+
+#### Swift
+```swift
+slider.accessibilityLabel = "Volume Slider"
+```
+
+### `-accessibilityActivate`
+ 
+ Our implementation closely resembles what UISlider does but it's not an exact match. On an `accessibilityActivate` we move one sixth of the amount between the current value and the midpoint value.
+
+### Minimum touch size
+
+Sliders currently respect the minimum touch size recomended by the Google Material spec [touch areas should be at least 48 points high and 48 wide](https://material.io/design/layout/spacing-methods.html#touch-click-targets). The height of the slider is set to 27 points so make sure there is sufficient space so that touches don't affect other elements.
```

Comment:
Should this be "at least 27 points"?  Does the Slider not work with `sizeToFit` or autolayout?

Reply:
Slider works with sizeToFit and autolayout but if a client put a button 8dp below or above they would have overlapping touch areas.

Reply:
This documentation seems reasonable, but as a team we should discuss our strategy around hit target versus UIView frames.

Reply:
I don't quite understand how setting a height to 27 means that there will be 48 points of space available for touch targets... Shouldn't the height be 48?

Reply:
We are using `pointInside` to detect touches outside the frame but within a 48 point touch area.